cmd/skopeo/copy.go

```diff
@@ -244,19 +252,21 @@ func (opts *copyOptions) run(args []string, stdout io.Writer) (retErr error) {
 
 	return retry.RetryIfNecessary(ctx, func() error {
 		manifestBytes, err := copy.Image(ctx, policyContext, destRef, srcRef, &copy.Options{
-			RemoveSignatures:      opts.removeSignatures,
-			SignBy:                opts.signByFingerprint,
-			SignPassphrase:        passphrase,
-			SignIdentity:          signIdentity,
-			ReportWriter:          stdout,
-			SourceCtx:             sourceCtx,
-			DestinationCtx:        destinationCtx,
-			ForceManifestMIMEType: manifestType,
-			ImageListSelection:    imageListSelection,
-			PreserveDigests:       opts.preserveDigests,
-			OciDecryptConfig:      decConfig,
-			OciEncryptLayers:      encLayers,
-			OciEncryptConfig:      encConfig,
+			RemoveSignatures:               opts.removeSignatures,
+			SignBy:                         opts.signByFingerprint,
+			SignPassphrase:                 passphrase,
+			SignByCosignPrivateKeyFile:     opts.signByCosignPrivateKey,
+			SignCosignPrivateKeyPassphrase: []byte(passphrase),
```

Comment:
This probably needs to ask for a passphrase on the terminal if nothing is specified (in both commands).

Reply:
Done.